### PR TITLE
quote commands in bin file

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

**Comment by josevalim** [comment]
I think we should quote just the command and not the modules. :)

**Comment by eksperimental** [comment]
done! ;-)

**Comment by josevalim** [comment]
Merged manually, thank you!

File: mix.lock [hunk 1]
```diff
@@ -1,3 +1,3 @@
 %{"earmark": {:hex, :earmark, "0.1.17"},
-  "hoedown": {:git, "git://github.com/hoedown/hoedown.git", "2a4cf17c70e6572ed42cdca3ea7ca3e768ed17be", []},
```
Comment (josevalim) [inline mix.lock]: You are using elixir master which now defaults to https. So you will need to be careful to not send the update mix.lock.

Comment (eksperimental) [inline mix.lock]: is there a way to avoid updating this file with a git config?

Comment (josevalim) [inline mix.lock]: If you want #git to ignore ONLY local changes to a file:

    $ git update-index --assume-unchanged path/to/some/file

[Via @hashnuke](https://twitter.com/HashNuke/status/612560420021256192).

Comment (eksperimental) [inline mix.lock]: :+1:

**Pull request closed without merging**